### PR TITLE
Linter Added

### PR DESCRIPTION
**Pull request by mfadhili**
Demo review

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+	background-size: cover;
+	background-position: center;
+}
+nav{
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	padding-top: 45px;
+	padding-left: 8%;
+	padding-right: 8%;
```
Comment (mfadhili) [inline css/style.css]: check indentation. Indentation should be 2spaces

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+	display: inline-block;
+	width: 100%;
+	padding: 14px 130px 14px 15px;
+	border: 2px solid #f9004d;
+	outline: none;
+	border-radius: 30px;
+}
+.newslatter form input:last-child{
+	position: absolute;
+	display: inline-block;
```
Comment (mfadhili) [inline css/style.css]: use proper formart for a starndard

File: index.html [hunk 3]
```diff
@@ -28,7 +28,7 @@ <h2 class="logo">Portfo<span>lio</span></h2>
 		<div class="content">
 			<h4>Hello, we are</h4>
 			<h1>Emery <span>Kroquet</span></h1>
-			<h1>And mfadhili <span>Gysa</span></h1>
+			<h1>And mfadhili <span>Gitau</span></h1>
 			<h3>We are Web Developer.</h3>
```
Comment (mfadhili) [inline index.html]: the changes are okay

File: css/style.css [hunk 4]
```diff
@@ -43,7 +43,7 @@ nav ul li a{
 }
 nav ul li a:hover{
```
Comment (mfadhili) [inline css/style.css]: fix the CSS file to conform to StyleLint

**Review by mfadhili — COMMENTED** [review]
Check these corrections please

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi, @mfadhili, 

Good job so far!
You are doing it well 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Comments are clear and easy to understand ✔️
- Good use of inline comments ✔️
- Correct use of summary ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Please 🙏🏽 instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it?
- [ ]  Please 🙏🏽 remember to be polite and clear in your comments - your job here is to help another person improve in a friendly way.
- [ ] Please 🙏🏽 explain the reason why a change is requested.
- [ ]  Kindly you can include emojis and gifs to make your code review more human-friendly 👽.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 5]
```diff
@@ -212,93 +212,104 @@ h5{
...
+  padding: 9px 22px;
+  border-radius: 30px;
+  transition: 0.4s; 
 }
-.card .button:hover{
-	background-color: transparent;
-	border: 2px solid #f9004d;
-	cursor: pointer;
+
+.card .button:hover {
```
Comment (mfadhili) [inline css/style.css:267]: You're doing great fixing the linter errors :+1: 
To better improve, try also adding a space before the curly brace.
Keep it up :1st_place_medal:

File: css/style.css [hunk 6]
```diff
@@ -169,48 +169,52 @@ button:hover{
...
+  position: relative;
+  overflow: hidden;
+  text-align: center;
 }
-.box{
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	min-height: 400px;
-}
```
Comment (mfadhili) [inline css/style.css]: you're almost there :clap: 
a few more corrections and you'll be done. :man_dancing: :woman_dancing: 
to make it better, Id suggest that you check the linters report.

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by GiuseppeTG** [comment]
Hi @mfadhili  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥇 : 
Congratulations! 🎉 

Alt Text

## Highlights
✅ You sounded more kind
✅ Good use of emojis
✅ Made suggestions instead of giving orders

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**